torntools/scripts/content/bazaar/ttBazaar.js

```diff
@@ -35,10 +35,10 @@ requireDatabase().then(() => {
 				let item_name = itemlist.items[item_id].name;
 
 				let found_item = false;
-				for (let item of doc.findAll("[class*='rowItems_'] [class*='item_']")) {
+				for (let item of doc.findAll("div[class*='item_']:not([class*='insensitive'])")) {
 					if (
-						item.find("[class*='name_']").innerText.trim() === item_name &&
-						item.find("[class*='price_']").innerText.replace("$", "").replace(/,/g, "") === item_price
+						item.find("[class*='name']").innerText.trim() === item_name &&
+						item.find("[class*='price']").innerText.replace(/["$,"]/g, "") === item_price
```

Comment:
Both of these selectors need a _ at the end.

I don't think the " in the regex is needed, as it would just try to filter out those.